doc/book/unit-testing.md

```diff
@@ -740,10 +745,7 @@ public function testExceptionIsThrownWhenGettingNonExistentAlbum()
         ->select(['id' => 123])
         ->willReturn($resultSet->reveal());
 
-    $this->setExpectedException(
-        RuntimeException::class,
-        'Could not find row with identifier 123'
-    );
+    $this->expectException(RuntimeException::class);
```

Comment:
I prefer the previous as:

- it works regardless of PHPUnit version
- it also tests the exception message